runtime/vm/CMakeLists.txt

```diff
@@ -21,23 +21,50 @@
 ################################################################################
 
 omr_add_tracegen(j9vm.tdf)
-j9vm_add_library(j9vm SHARED
-	OUTPUT_NAME j9vm${J9VM_VERSION_SUFFIX}
+
+set(interp_flags_to_remove) # list of flags removed when compiling the interpreter
+set(interp_new_flags) # flags to be added when compiling the interpreter
+
+if(OMR_ARCH_POWER)
+	if(OMR_TOOLCONFIG STREQUAL "xlc")
+		list(APPEND interp_flags_to_remove "-O3")
+		list(APPEND interp_new_flags "-O2" "-qdebug=lincomm:ptranl:tfbagg")
```

Comment:
This doesn't apply to pLinux BE, only AIX (although pLinux BE uses gcc for the interpreter).